decision/12.md

```diff
@@ -0,0 +1,46 @@
+---
+published: true
+layout: default-theme-wet-boew-en
+title: Design decision 12 - Double H1 on a single page
+description: Confirming the possibility to use two H1s on a single page and conform to accessibility requirements
+modified: 2020-06-30
+---
+
+## Summary
+
+* **Author:** @GormFrank
+* **Expert:** @duboisp, @bsouster, @namjohn920, @GormFrank
+* **Status:** Draft
+* **Last updated:** 2020-06-30
+* **History:**
+	* (2020-06-30) Initial draft
+
```

Comment:
Add a little bit of background on how to properly doing it. Like using it in combination of sectioning element as illustrated in the whatwg spec.